eotsmanager/cmd/eotsd/daemon/keys.go

```diff
@@ -37,6 +37,50 @@ func NewKeysCmd() *cobra.Command {
 		panic("failed to find keys add command")
 	}
 
+	listCmd := util.GetSubCommand(keysCmd, "list")
+	if listCmd == nil {
+		panic("failed to find keys list command")
+	}
+
+	listCmd.RunE = runCommandPrintAllKeys
+	listCmd.Flags().StringP(flags.FlagOutput, "o", flags.OutputFormatText, "Output format (text|json)")
```

Comment:
suggestion to remove this all good to use --output json
```suggestion
```

Reply:
it shouldn't since it works locally, maybe there is something weird with the test setup

Reply:
humm, I had removed and run the test and it passes `TestPrintEotsCmd`

Reply:
done at https://github.com/babylonlabs-io/finality-provider/pull/341